src/status_im2/contexts/quo_preview/buttons/slide_button.cljs

```diff
@@ -1,39 +1,57 @@
 (ns status-im2.contexts.quo-preview.buttons.slide-button
   (:require [quo2.core :as quo]
             [reagent.core :as reagent]
-            [status-im2.contexts.quo-preview.preview :as preview]))
+            [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]))
 
 (def descriptor
   [{:key     :size
     :type    :select
-    :options [{:key :size/s-48}
-              {:key :size/s-40}]}
+    :options [{:key :size-48}
+              {:key :size-40}]}
+   {:key     :type
+    :type    :select
+    :options [{:key :default}
+              {:key :danger}]}
    {:key  :disabled?
     :type :boolean}
+   {:key  :blur?
+    :type :boolean}
    (preview/customization-color-option {:key :color})])
 
-(defn view
+(defn f-view
   []
   (let [state     (reagent/atom {:disabled? false
                                  :color     :blue
-                                 :size      :size/s-48})
+                                 :size      :size-48})
         color     (reagent/cursor state [:color])
+        blur?     (reagent/cursor state [:blur?])
         complete? (reagent/atom false)]
     (fn []
+      (rn/use-effect (fn []
+                       (reset! complete? true)
+                       (js/setTimeout #(reset! complete? false) 50))
+                     [(:size @state)])
       [preview/preview-container
        {:state                     state
         :descriptor                descriptor
-        :component-container-style {:align-items :center}}
+        :component-container-style (when-not @blur? (:align-items :center))
+        :blur?                     @blur?
+        :show-blur-background?     true}
```

Comment:
If blur is only for the dark theme, should add the `blur-dark-only?` prop as well.

Reply:
imo, this feature would have benefitted from discussion before being merged/moved forward. This `blur-dark-only?` option ignores that there is also `blur-light-only` (at least I think there is some cases). Not to mention that there is many incorrect configurations of components possible on the preview screens. The button is a good example of this and a better alternative (imo) is to block incorrect configurations via disabling the descriptors for relevant options. This would of course benefit a lot if Mali gets added to the codebase. 

Having this discussion was mentioned here (although I'm not sure if the pr with this feature had got merged before, I assume it had so no big deal): 
https://github.com/status-im/status-mobile/pull/17271#issuecomment-1717450276
response  - https://github.com/status-im/status-mobile/pull/17271#issuecomment-1717517944
response  - https://github.com/status-im/status-mobile/pull/17271#issuecomment-1717914905

imo it should be commonplace to discuss with those interested in the team before we add new features to the preview screens as they tend to effect everyone, and with a team so large it takes a while to make later refactors once a feature starts getting used.